Objects/capsule.c

```diff
@@ -279,6 +303,29 @@ capsule_repr(PyObject *o)
 }
 
 
+static int
+capsule_traverse(PyCapsule *capsule, visitproc visit, void *arg)
+{
+    if (capsule->traverse_func) {
+        return capsule->traverse_func((PyObject*)capsule, visit, arg);
+    }
+    else {
+        return 0;
+    }
```

Comment:
`else` is unneeded here.

Reply:
Currently, the callback *can* be NULL. I added a check in my second PR to reject NULL callbacks.